README.dev.md

```diff
@@ -2,24 +2,19 @@
 
 ## Cutting a release
 
-* Make the desired changes to this repository. Most commonly, this involves updating the Docker image versions in `values.yaml` to match the tagged version you are releasing.
-* Run `./generate.sh`.
-* Open a PR and wait for buildkite to pass and for your changes to be approved, then merge and check out master.
-* Test this release on a fresh cluster cutting a real release. If this release includes changes not
-  yet deployed to dogfood (generally if it includes k8s config changes, rather than merely Docker
-  image changes), cut a test release (`LATEST=false ./release $TEST_VERSION`). Then go to the
-  `infrastructure` repository and follow the instructions in `datacenter/README.md` to update
-  dogfood to the test release.
-* Run `./release.sh $VERSION`.
-  * If this is a patch version to a previous major/minor version, use `LATEST=false ./release.sh
-    $VERSION`.
-  * If this is a release candidate, run `LATEST=false ./release {$VERSION}-pre${N}` where `N` starts
+- Make the desired changes to this repository. Most commonly, this involves updating the Docker image versions in `values.yaml` to match the tagged version you are releasing.
```

Comment:
Given we're updating these manually @ggilmore @nicksnyder -- when do our language server images get updated? I see that xlanGo, xlangJava, xlangTypescriptJavascript are not tagged with 2.10 and I worry we will end up forgetting to update these because of it.

Reply:
Right now all of our language servers are updated on a different release cycle than all of the other data center components. I can't think of a time where we made a breaking change in the language server (i.e. you can mix and match versions). I'm not opposed to the idea of versioning it along side the other components though.

Reply:
I am not worried about breaking changes. I am worried that data center users will not receive the latest updates to our language servers simply because we will forget to update the images here. This happened very often before, which was why @beyang had added the `update_docker_image_versions.py` script. But now that script is gone / not a part of our release process here, meaning it is trivial once again to forget to update these as part of cutting a release.

Reply:
(To be clear, I'm not saying we need to version them, I'm just saying we need this release process to *always* update them. Maybe that's as simple as adding instructions here to go to our CI and update each langserver version with the latest found there.)